distribution/docs/src/main/resources/_contents/_transformers/input-transformers-contents.adoc

```diff
@@ -0,0 +1,20 @@
+
+=== Included Input Transformers
+
+The following input transformers are available with a standard installation of ${branding}.
+
+<<_geojson_input_transformer,GeoJSON Input Transformer>>:: The GeoJSON input transformer is responsible for translating specific GeoJSON into a Catalog metacard.
+
+<<_gmd_transformer,GmdTransformer>>:: _description_
+
+<<_pdf_input_transformer,PdfInputTransformer>>:: _description_
+
+<<_pptx_input_transformer,PPTX Input Transformer>>:: The PPTX Input Transformer is the input transformer responsible for translating Microsoft PowerPoint (OOXML only) documents into a Catalog Metacard.
+
+<<_registry_transformer,RegistryTransformer>>:: _description_
```

Comment:
The Registry Transformer is responsible for creating Registry metacards from ebrim messages.